src/config.ts

```diff
@@ -0,0 +1,42 @@
+import { readFileSync, existsSync } from 'fs';
+import { load } from 'js-yaml';
+
+export default class Config {
+    public readonly token: string;
+    public readonly channelid: string;
+    public readonly color: number;
+    public readonly ownerid: string;
+    public readonly pgusername: string;
+    public readonly pgpassword: string;
+    public readonly pghost: string;
+    public readonly pgport: number;
+    public readonly pgdbname: string;
```

Comment:
Please use an interface for database, more info [here](https://www.typescriptlang.org/docs/handbook/interfaces.html)